### ISSUE TITLE
Login Background Image Never Applied

### ISSUE THREAD

**Issue by mrnoisytiger** [nextcloud/server#9306]
### Steps to reproduce
1. Try to add a background image in the Theme pane of Settings
2. Images uploads successfully
3. Go to log-in screen

### Expected behaviour
Background image should appear

### Actual behaviour
Only the defined "Color" shows for the entire background

### Server configuration

**Operating system**: Ubuntu Server 14.04LTS

**Web server:**: Apache

**Database:**: MySQL

**PHP version:**: 7.0

**Nextcloud version:** (see Nextcloud admin page): 13.0.1

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.6.1
  - admin_audit: 1.3.0
  - audioplayer: 2.2.5
  - bruteforcesettings: 1.0.3
  - comments: 1.3.0
  - dav: 1.4.6
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_downloadactivity: 1.2.0
  - files_external: 1.4.1
  - files_external_dropbox: 1.0.0
  - files_fulltextsearch: 0.6.0
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.2.1
  - files_retention: 1.2.0
  - files_rightclick: 0.6.0
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - fulltextsearch: 0.6.1
  - fulltextsearch_elasticsearch: 0.6.1
  - gallery: 18.0.0
  - guests: 0.5.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - metadata: 0.6.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - onlyoffice: 1.3.0
  - password_policy: 1.3.0
  - previewgenerator: 1.0.9
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - socialsharing_email: 1.0.3
  - socialsharing_facebook: 1.0.2
  - spreed: 3.1.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.1
  - theming_customcss: 1.0.0
  - twofactor_backupcodes: 1.2.3
  - twofactor_totp: 1.4.1
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - drawio
  - encryption
  - external
  - files_accesscontrol
  - files_automatedtagging
  - files_clipboard
  - ocr
  - unsplash
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "1": "homedrop.org",
            "2": "www.homedrop.org"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "overwrite.cli.url": "https:\/\/www.homedrop.org\/",
        "htaccess.RewriteBase": "\/",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "forwarded_for_headers": [
            "HTTP_X_FORWARDED",
            "HTTP_FORWARDED_FOR"
        ],
        "dbtype": "mysql",
        "version": "13.0.1.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "tls",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "LOGIN",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "maintenance": false,
        "appstore.experimental.enabled": true,
        "loglevel": 3,
        "trashbin_retention_obligation": "7, 14",
        "asset-pipeline.enabled": true,
        "updatechecker": false,
        "check_for_working_wellknown_setup": false,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "knowledgebaseenabled": false,
        "enable_previews": true,
        "preview_max_filesize_image": 100,
        "enabledPreviewProviders": [
            "OC\\Preview\\PNG",
            "OC\\Preview\\JPEG",
            "OC\\Preview\\GIF",
            "OC\\Preview\\BMP",
            "OC\\Preview\\XBitmap",
            "OC\\Preview\\MP3",
            "OC\\Preview\\TXT",
            "OC\\Preview\\MarkDown",
            "OC\\Preview\\PDF",
            "OC\\Preview\\TIFF",
            "OC\\Preview\\MSOfficeDoc",
            "OC\\Preview\\MSOffice2007",
            "OC\\Preview\\MSOffice2003",
            "OC\\Preview\\Photoshop"
        ],
        "updater.release.channel": "stable"
    }
}
```
</details>

**Are you using encryption:** No

### Client configuration
**Browser:**: Safari, Chrome

**Operating system:**: macOS High Sierra

**Comment by mrnoisytiger** [comment]
Using Inspect Element, it appears that this style is causing the issue:

``` css
#body-login, #firstrunwizard .firstrunwizard-header, #theming-preview {
    background-image: none !important;
    background-color: #082299;
}
```

Removing these styles causes the default background image to appear. However, the custom background image set still does not show.

**Comment by MorrisJobke** [comment]
cc @nextcloud/theming

**Comment by juliushaertl** [comment]
This is most likely a duplicate of https://github.com/nextcloud/server/issues/8888 which is fixed already in the upcoming 13.0.2.
